apps/challenges/migrations/0010_merge_20170108_0823.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```

Comment:
Oh I see, my bad. I think I missed this and forgot to ask Rishabh to just modify the previous migration file.